src/securityPolicy.c

```diff
@@ -296,6 +347,9 @@ security_policy_t policyForSignTxOutputAddressParams(
 
 		break;
 	}
+	case SIGN_TX_USECASE_MULTISIG:
+		DENY();
+		break;
```

Comment:
It'd be best if all cases used {} consistently. (And I'd welcome a blank line separating them.)

Reply:
If you wish so!